tutorial/README.rst

```diff
@@ -105,41 +132,49 @@ uses it to illustrate a range of framework features.
       #. Add a renewable-resource supply curve for the wind power plant,
          (:tut:`westeros/westeros_renewable_resource.ipynb`).
 
-   #. Variablity in energy supply and demand by adding sub-annual time steps,
-      e.g. winter and summer (:tut:`westeros/westeros_seasonality.ipynb`).
-   #. Using share constraints to depict policies, i.e. require renewables to
-      supply a certain share of total electricity generation
-      (:tut:`westeros/westeros_share_constraint.ipynb`).
-   #. Add a fossil-resource supply curve for the coal power plant,
-      (:tut:`westeros/westeros_fossil_resource.ipynb`).
-   #. Add-on technologies: Add the possibility of co-generation for the coal
-      power plant, by allowing it to produce heat via a passout-turbine
+   #. Sub-annual time resolution:
+
+      Represent variability in energy supply and demand by adding sub-annual
+      time resolution, e.g. winter and summer
+      (:tut:`westeros/westeros_seasonality.ipynb`).
+
+   #. Constraints:
+
+      #. Using share constraints to depict policies, i.e. require renewables to
+         supply a certain share of total electricity generation
+         (:tut:`westeros/westeros_share_constraint.ipynb`).
+      #. Add soft constraints for activity related dynamic constraints
+         (:tut:`westeros/westeros_soft_constraints.ipynb`
+
+   #. Add-on technologies:
+
+      Add the possibility of co-generation for the coal power plant, by
+      allowing it to produce heat via a passout-turbine
       (:tut:`westeros/westeros_addon_technologies.ipynb`).
-   #. Build the baseline scenario using data stored in xlsx files to populate sets and
-      parameters.
 
-      #. Export data to and Excel file and import the data to create a new scenario
-         (:tut:`westeros/westeros_baseline_using_xlsx_import_part1.ipynb`).
-      #. Import data from multiple excel files to create a new scenario
-         (:tut:`westeros/westeros_baseline_using_xlsx_import_part2.ipynb`).
+   #. Use parameters to represent the historical characteristics of the energy
+      system (:tut:`westeros/westeros_historical_new_capacity.ipynb`).
+
+#. Use other features of :mod:`message_ix` and :mod:`ixmp`:
 
-   #. Add soft constraints for activity related dynamic constraints
-      (:tut:`westeros/westeros_soft_constraints.ipynb`
+   #. ⭐ After the MESSAGE model has solved, use the :mod:`.message_ix.reporting`
+      module to ‘report’ results, e.g. do post-processing, plotting, and other
+      calculations (:tut:`westeros/westeros_report.ipynb`).
 
-   #. Explore historical parameters and their usage
-      (:tut:`westeros/westeros_historical_new_capacity.ipynb`).
-        
-#. After the MESSAGE model has solved, use the :mod:`.message_ix.reporting`
-   module to ‘report’ results, e.g. do post-processing, plotting, and other
-   calculations (:tut:`westeros/westeros_report.ipynb`).
+   #. ⭐ Build the baseline scenario using data stored in Excel files to
```

Comment:
```suggestion
   #. Build the baseline scenario using data stored in Excel files to
```

Reply:
Yes please go ahead! It's your PR. I'll approve once done.